WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -627,9 +629,11 @@ private class MigrationCell: WPTableViewCell {
         return stackView
     }()
 
-    private let logoImageView: UIImageView = {
+    private let migrationLogoImageView: UIImageView = {
         let imageView = UIImageView(image: .init(named: "wp-migration-welcome"))
         imageView.contentMode = .scaleAspectFit
+        imageView.heightAnchor.constraint(equalToConstant: LayoutConstants.imageSize.height).isActive = true
+        imageView.widthAnchor.constraint(equalToConstant: LayoutConstants.imageSize.width).isActive = true
```

Comment:
Moved these here because we want to be able to override the image view and the overriding image view has different size.